boot/assets.go

```diff
@@ -646,7 +647,8 @@ func (o *TrustedAssetsUpdateObserver) Canceled() error {
 		return fmt.Errorf("cannot write modeeenv: %v", err)
 	}
 
-	if err := resealKeyToModeenv(dirs.GlobalRootDir, o.model, o.modeenv); err != nil {
+	const expectReseal = true
+	if err := resealKeyToModeenv(dirs.GlobalRootDir, o.model, o.modeenv, expectReseal); err != nil {
```

Comment:
same question here, we pass true because we expect to need to reseal to the original assets because we were cancelled mid-way through an update correct?

Reply:
answered in the other one